<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QValueScaleDlg</class>
 <widget class="QWidget" name="QValueScaleDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>源范围</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>源范围：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_SourcecbBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>常量</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>地址</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="m_SMaxlineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LW0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_SMinlineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LW0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_sMaxAddress_EntryBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="m_sMinAddress_EntryBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_SMinspBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_SMaxspBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>显示范围</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_ShowcbBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>常量</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>地址</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>显示范围：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_ShowMaxlineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LW0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_ShowMinlineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LW0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_showMaxAddress_EntryBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="m_showMinAddress_EntryBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../resourse.qrc">
      <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_ShowMinspBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_ShowMaxspBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
